File: testtt/test1/test/src/model/Livraison.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

/**
 *
 * @author 21627
 */
public class Livraison {
     private int idLivraison,idCommande,idLivreur,prix;
     private String ville,etat,emailLivreur;
     

    public int getIdLivraison() {
        return idLivraison;
    }

    public Livraison() {
    }

    public Livraison(int idLivraison, int idCommande, int idLivreur, int prix, String ville, String etat, String emailLivreur) {
        this.idLivraison = idLivraison;
        this.idCommande = idCommande;
        this.idLivreur = idLivreur;
        this.prix = prix;
        this.ville = ville;
        this.etat = etat;
        this.emailLivreur = emailLivreur;
    }

    public Livraison(int idCommande, int idLivreur, int prix, String ville, String etat, String emailLivreur) {
        this.idCommande = idCommande;
        this.idLivreur = idLivreur;
        this.prix = prix;
        this.ville = ville;
        this.etat = etat;
        this.emailLivreur = emailLivreur;
    }

    
    public Livraison(String emailLivreur) {
        this.emailLivreur = emailLivreur;
    }

    public Livraison(String ville, String etat) {
        this.ville = ville;
        this.etat = etat;
    }

    public Livraison(int idCommande,int idLivreur,  int prix, String ville, String etat) {
        this.idLivreur = idLivreur;
        this.idCommande = idCommande;
        this.prix = prix;
        this.ville = ville;
        this.etat = etat;
    }
    
    public Livraison(int idCommande,String emaildLivreur,  int prix, String ville, String etat) {
        this.emailLivreur = emaildLivreur;
        this.idCommande = idCommande;
        this.prix = prix;
        this.ville = ville;
        this.etat = etat;
    }

    public Livraison(int prix, String ville) {
        this.prix = prix;
        this.ville = ville;
    }

    public Livraison(int idCommande, int prix, String ville, String etat) {
        this.idCommande = idCommande;
        this.prix = prix;
        this.ville = ville;
        this.etat = etat;
    }
    
   public Livraison(  int idLivreur, String ville, String etat) {
       
        this.idLivreur = idLivreur;
        this.ville = ville;
        this.etat = etat;
    }
   

    public Livraison(int idLivraison, int idCommande,int idLivreur,  int prix, String ville, String etat) {
        this.idLivraison = idLivraison;
        this.idCommande = idCommande;
         this.idLivreur = idLivreur;
        this.prix = prix;
        this.ville = ville;
        this.etat = etat;
    }

    @Override
    public String toString() {
        return "Livraison{" + "idLivraison=" + idLivraison + ",idCommande=" + idCommande + ", idLivreur=" + idLivreur + ",  prix=" + prix + ", ville=" + ville + ", etat=" + etat + '}';
    }

    public int getIdLivreur() {
        return idLivreur;
    }

    public void setIdLivreur(int idLivreur) {
        this.idLivreur = idLivreur;
    }


    

    public int getIdCommande() {
        return idCommande;
    }

    public void setIdCommande(int idCommande) {
        this.idCommande = idCommande;
    }

    public int getPrix() {
        return prix;
    }

    public void setPrix(int prix) {
        this.prix = prix;
    }

    public String getVille() {
        return ville;
    }

    public void setVille(String ville) {
        this.ville = ville;
    }

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }
     

    
}

File: src/Entity/Livraison.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Livraison
 *
 * @ORM\Table(name="livraison", indexes={@ORM\Index(name="fk_commande", columns={"idCommande"}), @ORM\Index(name="fk_livreur", columns={"idLivreur"})})
 * @ORM\Entity
 */
class Livraison
{
    /**
     * @var int
     *
     * @ORM\Column(name="idLivraison", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idlivraison;

    /**
     * @var int
     *
     * @ORM\Column(name="idCommande", type="integer", nullable=false)
     */
    private $idcommande;

    /**
     * @var int
     *
     * @ORM\Column(name="idLivreur", type="integer", nullable=false)
     */
    private $idlivreur;

    /**
     * @var int
     *
     * @ORM\Column(name="prix", type="integer", nullable=false)
     */
    private $prix;

    /**
     * @var string
     *
     * @ORM\Column(name="ville", type="string", length=80, nullable=false)
     */
    private $ville;

    /**
     * @var string
     *
     * @ORM\Column(name="etat", type="string", length=80, nullable=false, options={"default"="En cours"})
     */
    private $etat = 'En cours';

    public function getIdlivraison(): ?int
    {
        return $this->idlivraison;
    }

    public function getIdcommande(): ?int
    {
        return $this->idcommande;
    }

    public function setIdcommande(int $idcommande): self
    {
        $this->idcommande = $idcommande;

        return $this;
    }

    public function getIdlivreur(): ?int
    {
        return $this->idlivreur;
    }

    public function setIdlivreur(int $idlivreur): self
    {
        $this->idlivreur = $idlivreur;

        return $this;
    }

    public function getPrix(): ?int
    {
        return $this->prix;
    }

    public function setPrix(int $prix): self
    {
        $this->prix = $prix;

        return $this;
    }

    public function getVille(): ?string
    {
        return $this->ville;
    }

    public function setVille(string $ville): self
    {
        $this->ville = $ville;

        return $this;
    }

    public function getEtat(): ?string
    {
        return $this->etat;
    }

    public function setEtat(string $etat): self
    {
        $this->etat = $etat;

        return $this;
    }


}
